docs/DevGuide.ManagingUsers.md

```diff
@@ -0,0 +1,137 @@
+4. Click on the **+** sign at the end if the users list. The **Add User** dialog is displayed and a password is generated for the user.
+   <img src="./_Images/old-ui-add-user.png">
+1. From the **Password** field, copy and save the password value for your own information.
+2. In the **Username** field, type the new username.
+2. In the **Permissions** field, edit the **paths** field as required.
+3. Click the **Create** button. The new user's FileInfoton is created.
+
+>**Note**: The **digest** and **digest2** values are not visible in the dialog, but are added to the FileInfoton automatically. 
+
+## Editing User Details ##
+
+To edit user details, follow the same procedure as for creating a new user. This overrides the existing FileInfoton with your new user details content.
+
+<a name="hdrGenPassword"></a>
+## Generating User Passwords ##
+
+You can use the [generate-password API](API.Auth.GeneratePassword.md) to generate a random password and its encoded digest value. 
+
+## Changing User Passwords ##
+
+You can use the [change-password API](API.Auth.GeneratePassword.md) to change an existing password.
+
+## Deleting Users ##
+
+To delete a user, simply delete the FileInfoton with the user's details, then call the [Invalidate Cache API](API.Auth.InvalidateCache.md) to make the deletion effective immediately.
+
+<a name="hdrRoles"></a>
+## Managing User Roles ##
+
+You may want to define user roles that confer a particular set of access permissions on users with those roles, to be applied to several users. You can do this by creating FileInfotons with the same **paths** field as described in [Creating New Users](#hdrCreateUsers) (no other fields are required).
+
+If a user's FileInfoton has the **roles** field, then the user takes on the permissions of those roles. For example, if user jsmith has "roles":["Editor"] in his FileInfoton, and there is a ```/meta/auth/roles/Editor``` FileInfoton which allows write permission on certain paths, then jsmith can write to those paths, even though they're not mentioned in his personal **paths** field.
+
+>**Note**: If there's a contradiction between personal permissions and role-based permissions, the personal permissions take precedence.
+
+**To create a role FileInfoton:**
+
+1. Login to CM-Well with an admin user. A token with admin permissions is returned in the response.
+1. Create the JSON body with role paths and permissions as required. See [paths field](#tableFields).
+1. Upload the JSON body you created to a FileInfoton named <rolename>, under the folder ```/meta/auth/roles```. For example:
+
+    ```curl cmwell/meta/auth/roles/Editor -H "X-CM-Well-Type:File" -H "Content-Type:application/json" -H "X-CM-Well-Token:<AdminToken>" --data-binary @roleDetails.json```
+
+1. In order for the new role information to take effect, refresh the CM-Well authentication cache using the following command:
+
+    ```\<cm-well-host\>/_auth?op=invalidate-cache -H "X-CM-Well-Token:<AdminToken>"```
```

Comment:
no need to escape `<` and `>` with a backslash, in case when whole thing is wrapped by ` ``` `s. Just GitHub Markdown things... i.e. correct line is ```<cm-well-host>/_auth?op=invalidate-cache -H "X-CM-Well-Token:<AdminToken>"```